charts/rancher-gke-operator/1.0.000/questions.yaml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.7
```

Comment:
Min should be `2.5.7-rc1`, I believe the `rc1` is arbitrary but it needs some sort of pre-release so it works on those.